pkg/kubectl/cmd/util/openapi/openapi_cache_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package openapi
+package openapi_test
```

Comment:
Why change the package name? Is this because trying to match the package name of openapi_suite_test.go?
I found other pkg usually the same name for the test.